src/main/java/com/minecolonies/core/colony/buildings/modules/MinimumStockModule.java

```diff
@@ -57,8 +57,9 @@ public class MinimumStockModule extends AbstractBuildingModule implements IMinim
     private int minimumStockSize()
     {
         final double increase = 1 + building.getColony().getResearchManager().getResearchEffects().getEffectStrength(MINIMUM_STOCK);
+        final int buildingLevelOr1 = building.getMaxBuildingLevel() != 0 ? building.getBuildingLevel() : 1;
 
-        return (int) (building.getBuildingLevel() * STOCK_PER_LEVEL * increase);
+        return (int) (buildingLevelOr1 * STOCK_PER_LEVEL * increase);
```

Comment:
Should maybe be a buildingLevel or maxlevel = 5 (postbox/stash should be maxlevel by default)

Reply:
This is why I said make a specific override of the min stock module, create a method that returns the stock level, and stock per level, and let the postbox override these with different values (not building level)

Reply:
I think that a special case is allowed to be special when there's only one of it, but if there's another special case then it's worth refactoring + generalizing.  As it is, there's the general case of hut blocks with levels, and the special case of hut blocks without levels (just postbox), so I think it's fine to leave as is.  

If you decide you want to make it affected differently by research or something, then you should extend the min stock module and move stuff around.